doc/common/Cadl-Project-Scripts.md

```diff
@@ -0,0 +1,139 @@
+# Integrating with cadl sync and generate scripts
+
+There are 2 common scripts provided for each language to be able to generate from within the language
+repo and use the remote cadl definition in the spec repo.
+
+## One time language repo setup
+
+There are 4 things that these two scripts expect are set up in your language repo before they will run correctly.
+
+### cadl-location.yaml
```

Comment:
`cadl-location.yaml` is a per service setup. So, it should be moved out of the `One time language repo setup`.